### DISCUSSION TITLE
Multiple Checkboxes?

### DISCUSSION THREAD

**Discussion by stramomike** [TriliumNext/Trilium#4233] [Q&A] [answered]
I am pretty new to trilium, and I am wondering if there is a way to have multiple checkboxes on a single line.

Essentially, I'm wanting something like:
Level 1   O O O O
Level 2   O O O
Level 3   O

Where the O's are checkable. Is this possible?

Thanks for your help!

**Comment by sottey** [comment]
Hope this helps. The answer is technically "yes", but I am guessing you want more functionality than just *showing* the stuff.

![image](https://github.com/zadam/trilium/assets/8494620/61fa740c-e2f3-4129-a9ae-dc4131b15415)

Here is the html to put into a Code-> HTML note:
`
Level 1 <input type="checkbox"> </input><input type="checkbox"> </input><input type="checkbox"> </input><input type="checkbox"> </input><br/>
Level 2 <input type="checkbox"> </input><input type="checkbox"> </input><input type="checkbox"> <br/>
Level 3 <input type="checkbox"> </input>
`

**Reply by stramomike** [reply]
This does seem useful!
Rendering seems to just display the code and not actually "run" it for me. Is there a step I'm missing?

**Reply by sottey** [reply, marked as answer]
The note that contains the code should be in a separate note. Then, the rendering note should be set to type "renderNote" and the renderNote value should be the ID of the child note. (In the earlier post I said "name of the note". Sorry about that)

So:
Note: Code html
Note Type: Code -> HTML
Contents: HTML you want to render

Note: My View
Note Type: RenderNote
Attributes: ~renderNote="[ID of Code html]"

Hope that helps!

**Reply by stramomike** [reply]
Thanks!
I was actually copying and pasting directly from this webpage to trilium, and I was grabbing extra bits of html from the comment box without realizing it.
This will do the trick! I feel like to do much more than this would essentially mean I'm making a full website for this purpose, which I don't need for just a few character sheets.

**Comment by zadam** [comment]
Checkboxes in CKEditor are technically lists with each checkbox being one list item. I don't think this would be possible without hacks.

**Comment by PenguinBartender** [comment]
The work-around I've found (on PC) is to use emojis for empty box ⏹️ and checked box ✅

it's a relatively easy keyboard shortcut: windows-key + period
